osu.Game.Rulesets.Catch/Edit/CatchHitObjectInspector.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Screens.Edit.Compose.Components;
+
+namespace osu.Game.Rulesets.Catch.Edit
+{
+    public partial class CatchHitObjectInspector(CatchDistanceSnapProvider snapProvider) : HitObjectInspector
+    {
+        protected override void AddInspectorValues(HitObject[] objects)
+        {
+            base.AddInspectorValues(objects);
+
+            if (objects.Length > 0)
+            {
+                HitObject firstSelectedHitObject = objects.MinBy(ho => ho.StartTime)!;
+                HitObject lastSelectedHitObject = objects.MaxBy(ho => ho.GetEndTime())!;
+
+                HitObject? precedingObject = EditorBeatmap.HitObjects.LastOrDefault(ho => ho.GetEndTime() < firstSelectedHitObject.StartTime);
+                HitObject? nextObject = EditorBeatmap.HitObjects.FirstOrDefault(ho => ho.StartTime > lastSelectedHitObject.GetEndTime());
+
+                if (precedingObject != null || nextObject != null)
+                {
+                    AddHeader("Snapping");
+                }
+
+                if (precedingObject != null)
+                {
+                    double previousSnap = snapProvider.ReadCurrentDistanceSnap(precedingObject, firstSelectedHitObject);
+                    AddValue($"Previous: {previousSnap:#,0.##}x");
+                }
+
+                if (nextObject != null)
+                {
+                    double nextSnap = snapProvider.ReadCurrentDistanceSnap(lastSelectedHitObject, nextObject);
+                    AddValue($"Next: {nextSnap:#,0.##}x");
+                }
```

Comment:
I'm weirded out by this formatting. osu! does this:

![1732010127](https://github.com/user-attachments/assets/cf15f75d-9567-4ac2-8cae-45cf3d21003e)

Can we use the same but change the units to the `#.##x` relative thing? I'd rather be consistent here.

Reply:
I intially made it consistent to how the position is shown, so like this:
![](https://puu.sh/KiYi8/9bba5c5ddb.png)
But consistency is probably better here 👍